presto-docs/src/main/sphinx/presto_cpp/properties.rst

```diff
@@ -0,0 +1,140 @@
+===============================
+Presto C++ Properties Reference
+===============================
+
+This section describes Presto C++ configuration properties.
+
+The following is not a complete list of all configuration and
+session properties, and does not include any connector-specific
+catalog configuration properties. For information on catalog 
+configuration properties, see :doc:`Connectors </connector/>`.
+
+.. contents::
+    :local:
+    :backlinks: none
+    :depth: 1
+
+Coordinator Properties
```

Comment:
@steveburnett : The co-ordinator properties have to be set exactly as specified. But the worker configs can be altered from the defaults. Do you feel we should document them separately so that the difference is made clear to the users ?

Reply:
I think the statement below makes it clear that the coordinator configs have to be set as specified?

Reply:
@aditi-pandit, the worker configs are documented separately in the Worker Properties topic. But I think I can make it more clear that the coordinator properties have to be set exactly as specified. Thanks!